### ISSUE TITLE
Unable to run tests locally

### ISSUE THREAD

**Issue by russcam** [PerfDotNet/BenchmarkDotNet#168]
I'm unable to build the project locally to run either classic or dnx tests.

running runDnxAndCoreTests.cmd I get

> System.ArgumentException: More than one runtime.json file has declared imports for 'win7-x86'
Parameter name: runtimeName
   at Microsoft.Dnx.Tooling.RestoreCommand.FindRuntimeDependencies(String runtimeName, List`1 runtimeFiles, Dictionary`2 effectiveRuntimeSpecs, HashSet`1 allRuntimeNames, Func`2 circularImport)
   at Microsoft.Dnx.Tooling.RestoreCommand.FindRuntimeDependencies(String runtimeName, List`1 runtimeFiles, Dictionary`2 effectiveRuntimeSpecs, HashSet`1 allRuntimeNames)
   at Microsoft.Dnx.Tooling.RestoreCommand.<RestoreForProject>d__69.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Dnx.Tooling.RestoreCommand.<>c__DisplayClass68_0.<<Execute>b__2>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   at Microsoft.Dnx.Tooling.RestoreCommand.<Execute>d__68.MoveNext()
----------
Restore failed
More than one runtime.json file has declared imports for 'win7-x86'
Parameter name: runtimeName

I suspect this may be to do with running dnx and dotnet cli side by side (I still need dnx for the moment).

running runClassicTests.cmd I get

> "D:\dev\git\BenchmarkDotNet\BenchmarkDotNet.sln" (build target) (1) ->
"D:\dev\git\BenchmarkDotNet\BenchmarkDotNet.Samples.FSharp\BenchmarkDotNet.Samples.FSharp.fsproj.metaproj" (default
target) (16) ->
"D:\dev\git\BenchmarkDotNet\BenchmarkDotNet.Samples.FSharp\BenchmarkDotNet.Samples.FSharp.fsproj" (default target) (
17) ->
(ResolveNuGetPackageAssets target) ->
  C:\Program Files (x86)\MSBuild\Microsoft\NuGet\Microsoft.NuGet.targets(109,5): error : Your project is not referen
cing the ".NETFramework,Version=v4.5.2" framework. Add a reference to ".NETFramework,Version=v4.5.2" in the "framewo
rks" section of your project.json, and then re-run NuGet restore. [D:\dev\git\BenchmarkDotNet\BenchmarkDotNet.Sample
s.FSharp\BenchmarkDotNet.Samples.FSharp.fsproj]

was wondering if anyone else had come across this problem?

**Comment by AndreyAkinshin** [comment]
@russcam, just remove the `BenchmarkDotNet.Samples.FSharp.Core` folder, it helps. You don't need this folder.

@adamsitnik should fix it.

**Comment by russcam** [comment]
Removed all the FSharp projects for the moment and also

```c#
[Fact]
public void FSharpIsSupported()
{
    CanCompileAndRun<FSharpBenchmark.Db>();
 }
```

from `BenchmarkDotNet.IntegrationTests.Classic.ReferenceTests` and can now run them for the moment :+1:

**Comment by adamsitnik** [comment]
I think that it has to do with the toolchains, either dnx or dotnet cli fault. Same issue was reported [here](https://github.com/Microsoft/visualfsharp/issues/832)

Could you create a new folder and run following commands:
```
dotnet net --lang f#
dotnet restore
dotnet run
```
Does it work?

If so, then does `dnu restore` works for it as well?

And could you also give me your `dnx --version` and `dotnet --version` output?

**Comment by AndreyAkinshin** [comment]
```
D:\Temp\sandbox
λ dotnet new --lang f#
Created new F# project in D:\Temp\sandbox.

D:\Temp\sandbox
λ ls
NuGet.Config*  Program.fs*  project.json*

D:\Temp\sandbox
λ dotnet restore
log  : Restoring packages for D:\Temp\sandbox\project.json...
info :   GET https://dotnetmyget.blob.core.windows.net/artifacts/dotnet-core/nuget/v3/flatcontainer/microsoft.netcore.platforms/index.json
info :   GET https://api.nuget.org/v3-flatcontainer/microsoft.netcore.platforms/index.json
info :   OK https://api.nuget.org/v3-flatcontainer/microsoft.netcore.platforms/index.json 518ms
info :   GET https://myget-2e16.kxcdn.com/artifacts/fsharp-daily/nuget/v3/flatcontainer/microsoft.netcore.platforms/index.json
info :   OK https://dotnetmyget.blob.core.windows.net/artifacts/dotnet-core/nuget/v3/flatcontainer/microsoft.netcore.platforms/index.json 1107ms
info :   NotFound https://myget-2e16.kxcdn.com/artifacts/fsharp-daily/nuget/v3/flatcontainer/microsoft.netcore.platforms/index.json 424ms
info : Committing restore...
log  : Restore completed in 6396ms.

NuGet Config files used:
    D:\Temp\sandbox\NuGet.Config
    D:\Temp\NuGet.Config
    C:\Users\akinshin\AppData\Roaming\NuGet\NuGet.Config

Feeds used:
    https://dotnet.myget.org/F/dotnet-core/api/v3/index.json
    https://api.nuget.org/v3/index.json
    https://www.myget.org/F/fsharp-daily/api/v3/index.json

D:\Temp\sandbox
λ dotnet run
Compiling sandbox for DNXCore,Version=v5.0

Compilation succeeded.
    0 Warning(s)
    0 Error(s)

Time elapsed 00:00:08.3142594


Hello World!
[||]

D:\Temp\sandbox
λ dnu restore
Microsoft .NET Development Utility Clr-x86-1.0.0-rc1-16609

  CACHE https://dotnet.myget.org/F/dotnet-core/api/v3/index.json
  CACHE https://api.nuget.org/v3/index.json
  CACHE https://www.myget.org/F/fsharp-daily/api/v3/index.json
Restoring packages for D:\Temp\sandbox\project.json

// *** Big log ***

----------
System.ArgumentException: More than one runtime.json file has declared imports for 'win7-x86'
Parameter name: runtimeName
   at Microsoft.Dnx.Tooling.RestoreCommand.FindRuntimeDependencies(String runtimeName, List`1 runtimeFiles, Dictionary`2 effectiveRuntimeSpecs, HashSet`1 allRuntimeNames, Func`2 circularImport)
   at Microsoft.Dnx.Tooling.RestoreCommand.FindRuntimeDependencies(String runtimeName, List`1 runtimeFiles, Dictionary`2 effectiveRuntimeSpecs, HashSet`1 allRuntimeNames)
   at Microsoft.Dnx.Tooling.RestoreCommand.<RestoreForProject>d__69.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Dnx.Tooling.RestoreCommand.<>c__DisplayClass68_0.<<Execute>b__2>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   at Microsoft.Dnx.Tooling.RestoreCommand.<Execute>d__68.MoveNext()
----------
Restore failed
More than one runtime.json file has declared imports for 'win7-x86'
Parameter name: runtimeName

NuGet Config files used:
    C:\Users\akinshin\AppData\Roaming\NuGet\nuget.config
    D:\Temp\nuget.config
    D:\Temp\sandbox\nuget.config

Feeds used:
    https://dotnetmyget.blob.core.windows.net/artifacts/dotnet-core/nuget/v3/flatcontainer/
    https://api.nuget.org/v3-flatcontainer/
    https://myget-2e16.kxcdn.com/artifacts/fsharp-daily/nuget/v3/flatcontainer/

D:\Temp\sandbox
λ dotnet --version
.NET Command Line Tools (1.0.0-beta-001598)

Product Information:
 Version:     1.0.0-beta-001598
 Commit Sha:  7582649f88

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.10586
 OS Platform: Windows
 Runtime Id:  win10-x64

D:\Temp\sandbox
λ dnx --version
Microsoft .NET Execution environment
 Version:      1.0.0-rc1-16609
 Type:         Clr
 Architecture: x86
 OS Name:      Windows
 OS Version:   10.0
 Runtime Id:   win10-x86
```

**Comment by adamsitnik** [comment]
Fixed with the migration to RC2 #187
